lib/binding.ts

```diff
@@ -59,7 +64,8 @@ export class CanBridge {
     initializeNotifier: () => void;
     waitForNotifierAlarm: (time:number) => Promise<number>;
     stopNotifier: () => void;
-    writeDfuToBin: (dfuFileName:string, binFileName:string) => Promise<number>;
+    writeDfuToBin: (dfuFileName:string, binFileName:string, elementIndex: number | undefined) => Promise<number>;
```

Comment:
```suggestion
    writeDfuToBin: (dfuFileName:string, binFileName:string, elementIndex?: number) => Promise<number>;
```

The way you had it, callers had to manually pass in `undefined`, instead of being able to leave off the parameter completely (which would still be a breaking change).